pages/shared/data/roadshow.yaml

```diff
@@ -16,70 +18,103 @@ nextstop:
 
 past:
   - title: NAIROBI
-    date: MAR 22, 2019  
+    coords: 65x33
+    date: MAR 22, 2019
   - title: LAGOS
-    date: MAR 19, 2019  
+    coords: 54x32
+    date: MAR 19, 2019
   - title: CAPE TOWN
-    date: MAR 15, 2019  
+    coords: 59x46
+    date: MAR 15, 2019
   - title: JOHANNESBURG
-    date: MAR 12, 2019  
+    coords: 63x43
+    date: MAR 12, 2019
   - title: HO CHI MINH CITY
+    coords: 88x24
     date: DEC 14, 2018
   - title: TORONTO
+    coords: 24x12
     date: AUG 30, 2017
   - title: SINGAPORE
+    coords: 86x29
     date: OCT 6, 2017
   - title: JAKARTA
```

Comment:
Singapore and Jakarta appear to both be on the same island on this map. I'm not sure what all the little stylized islands are supposed to be, but it would be nice if Singapore were somehow part of the mainland. As it's on a peninsula.

Reply:
Singapore is now on the mainland